src/v2/Apps/Artist/Routes/WorksForSale/ArtistWorksForSaleRoute.tsx

```diff
@@ -20,6 +22,20 @@ const ArtistWorksForSaleRoute: React.FC<ArtistWorksForSaleRouteProps> = ({
     artist?.counts?.forSaleArtworks!,
     true
   )
+  const { match } = useRouter()
+  const { scrollTo, isMatchMediaParsed } = useScrollTo({
+    selectorOrRef: "#jump--artworkFilter",
+    behavior: "smooth",
+    offset: 10,
+  })
+
+  useEffect(() => {
+    if (isMatchMediaParsed && match?.location?.query?.search_criteria_id) {
+      requestAnimationFrame(() => {
```

Comment:
Why wrap this in a RAF?

Reply:
To make `scrollTo` work for Safari

Reply:
Why doesn't it work in Safari, and what about RAF makes it work? Does `setTimeout(() => {}, 0)` accomplish the same thing?

Reply:
> Does setTimeout(() => {}, 0) accomplish the same thing?

Yep

Reply:
> Why doesn't it work in Safari, and what about RAF makes it work?

Unfortunately, I could not figure out the reason why `scrollTo` doesn't work in Safari if used in `useEffect`

For example, if we use this code, it will work fine in Google Chrome, but it will not work in Safari
```javascript
React.useEffect(() => {
  window.scrollTo({
    top: 1000,
    behavior: "smooth",
  })
})
```

### Google Chrome
https://user-images.githubusercontent.com/3513494/143281470-c104c4f7-b090-4d1f-806f-1c63bec0a05b.mp4

### Safari
https://user-images.githubusercontent.com/3513494/143281514-e70690a5-46b1-43c4-ab88-4fe7d795d6e8.mp4

Reply:
Got it — I'd just add a comment explaining that. Anything that's not immediately obvious, ya kno?

Also, I'm curious if `useLayoutEffect` makes any difference?

Reply:
Unfortunately, `useLayoutEffect` doesn't work either

Reply:
👍